GMCMultiImage/GMCZoomingMultiImageView.m

```diff
@@ -212,29 +212,31 @@ - (void)updateImage {
                     }
                     
                     if (!error) {
-                        dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0), ^{
+                        dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
Makes sense to me. In that case, can you take a look at the other 2 calls to `fetchImageWithCompletionBlock` below? The error case of the 3rd one makes a UIKit call, that's all I could identify from visual inspection.